docs/make.jl

```diff
@@ -1,7 +1,7 @@
 using Dionysos
 using Documenter, Literate
 
-const EXAMPLES_DIR = joinpath(@__DIR__, "..", "examples")
+const EXAMPLES_DIR = joinpath(@__DIR__, "src", "examples")
 const OUTPUT_DIR   = joinpath(@__DIR__, "src/generated")
 
 include(joinpath(EXAMPLES_DIR, "all_examples.jl"))
```

Comment:
Replace with `const EXAMPLES = readdir(EXAMPLES_DIR)`

Reply:
The include, you mean?

Reply:
Or in the all_examples.jl file?

Reply:
I think you meant the include, right? If so, do I get rid of all_examples.jl?

Reply:
Yes, remove the include :)